lib/inv_ca_cg.cpp

```diff
@@ -12,37 +12,35 @@
 
 namespace quda {
 
-  enum Basis {
-    POWER_BASIS,
-    INVALID_BASIS
-  };
-
-  const static Basis basis = POWER_BASIS;
-
   CACG::CACG(DiracMatrix &mat, DiracMatrix &matSloppy, SolverParam &param, TimeProfile &profile)
     : Solver(param, profile), mat(mat), matSloppy(matSloppy), init(false),
-      W(nullptr), C(nullptr), alpha(nullptr), beta(nullptr), phi(nullptr), rp(nullptr),
+      lambda_init(false), basis(param.ca_basis),
+      Q_AQandg(nullptr), Q_AS(nullptr), alpha(nullptr), beta(nullptr), rp(nullptr),
       tmpp(nullptr), tmpp2(nullptr), tmp_sloppy(nullptr), tmp_sloppy2(nullptr) { }
```

Comment:
clang-format suggested the following change:


```
-  CACG::CACG(DiracMatrix &mat, DiracMatrix &matSloppy, SolverParam &param, TimeProfile &profile)
-    : Solver(param, profile), mat(mat), matSloppy(matSloppy), init(false),
-      lambda_init(false), basis(param.ca_basis),
-      Q_AQandg(nullptr), Q_AS(nullptr), alpha(nullptr), beta(nullptr), rp(nullptr),
-      tmpp(nullptr), tmpp2(nullptr), tmp_sloppy(nullptr), tmp_sloppy2(nullptr) { }
+  CACG::CACG(DiracMatrix& mat, DiracMatrix& matSloppy, SolverParam& param, TimeProfile& profile)
+      : Solver(param, profile)
+      , mat(mat)
+      , matSloppy(matSloppy)
+      , init(false)
+      , lambda_init(false)
+      , basis(param.ca_basis)
+      , Q_AQandg(nullptr)
+      , Q_AS(nullptr)
+      , alpha(nullptr)
+      , beta(nullptr)
+      , rp(nullptr)
+      , tmpp(nullptr)
+      , tmpp2(nullptr)
+      , tmp_sloppy(nullptr)
+      , tmp_sloppy2(nullptr)
+  {
+  }
```

<!-- identifier: clang-format -->
<!-- File: lib/inv_ca_cg.cpp -->